Tests/DatadogTests/Datadog/RUMMonitorTests.swift

```diff
@@ -445,7 +445,12 @@ class RUMMonitorTests: XCTestCase {
             directory: temporaryDirectory,
             dependencies: .mockWith(
                 userInfoProvider: .mockWith(
-                    userInfo: UserInfo(id: "abc-123", name: "Foo", email: "foo@bar.com")
+                    userInfo: UserInfo(
+                        id: "abc-123",
+                        name: "Foo",
+                        email: "foo@bar.com",
+                        extraInfo: ["extra_key": "extra_value"]
```

Comment:
Because `extraInfo` encoding depends on the feature (Logging and RUM supports multiple `Encodable` types, but Tracing encodes all values as `String`), it seems better to mix few types in this test:
```swift
extraInfo: ["extra_string": "string_value", "extra_int": 42, "extra_bool": true]
```
and document the expected encoding result in test assertions.